docs/index.yml

```diff
@@ -278,7 +281,7 @@ additionalContent:
             text: Get started in 5 minutes
           - url: iot/tutorials/blink-led.md
             text: Blink an LED
-          - url: https://aka.ms/IoTNet101
+          - url: <https://aka.ms/IoTNet101>
```

Comment:
May need confirmation on staging that usage of markdown link syntax still works in YAML.

Reply:
This doesn't work, and correctly triggers a build warning:
![image](https://user-images.githubusercontent.com/7679720/112149336-dc348f80-8bac-11eb-957e-9faa6fdda546.png)

Reply:
It's markdownlint that does that and I just missed cleaning it up after running it to do cleanup. ;)